src/components/books.jsx

```diff
@@ -1,30 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
-import '../index.css';
+import logo from './Logo.png';
 
 const Book = ({ id, title, author }) => {
+  const [change, setChange] = useState(15);
   const dispatch = useDispatch();
-
   const handleRemoveBook = () => {
     dispatch(deleteBook(id));
   };
+  function handleChange() {
+    setChange(change + 1);
+  }
 
   return (
-    <section className="book">
-      <div>
-        <h3 className="book-title">{title}</h3>
-        <p className="book-author">{author}</p>
+    <section className="book flex flex-row justify-between">
+      <div className="book-right-part w-1/2 flex flex-col justify-evenly">
+        <div>
+          <h3 className="book-title font-serif font-bold text-xl mb-1">{title}</h3>
+          <p className="book-author text-sm font-serif text-blue font-extralight">{author}</p>
+        </div>
+        <div className="flex justify-evenly w-1/2 items-center text-blue font-serif text-base mt-1">
+          <a href="###" onClick={handleRemoveBook}>
+            commens
+          </a>
+          <a href="###" onClick={handleRemoveBook}>
+            remove
+          </a>
+          <a href="###" onClick={handleRemoveBook}>
+            edit
+          </a>
+        </div>
```

Comment:
- [ ] If you pay a closer attention on the book item you will noticed that , the `comment` (where you wrote commens 😄 😄 ) , is not aligned with the book title, Kindly fix that to meet the requirements 👍 

### You design 
![Capture1](https://github.com/mhnazary/bookstore/assets/80458199/8ec6f157-9b09-4799-89bf-a9863319c8c8)

### Template design
![Capture11](https://github.com/mhnazary/bookstore/assets/80458199/e1ec3288-b65c-4825-9877-7553c420497a)